gcloud_requests/proxy.py

```diff
@@ -76,31 +64,30 @@ def __init__(self, credentials=None, logger=None):
     def __del__(self):
         _credentials_watcher.unwatch(self.credentials)
 
-    def request(self, uri, method="GET", body=None, headers=None, redirections=5, connection_type=None, retries=0, refresh_attempts=0):   # noqa
+    def request(self, method, url, data=None, headers=None, retries=0, refresh_attempts=0, **kwargs):
         session = self._get_session()
         headers = headers.copy() if headers is not None else {}
         auth_request = AuthRequest(session=session)
         retry_auth = partial(
             self.request,
-            uri=uri, method=method,
-            body=body, headers=headers,
-            redirections=redirections,
-            connection_type=connection_type,
+            url=url, method=method,
```

Comment:
It doesn't matter since they're being sent as keyword args, but why is the order here different than in the signature?

Reply:
Only because I was just updating `uri` to be `url` here, but like you said it doesn't matter because the keywords are explicit.